conf/base_url_conf.py

```diff
@@ -2,4 +2,5 @@
 Conf file for base_url
 """
 base_url = "https://qxf2.com/"
```

Comment:
Can you rename `base_url` as `ui_base_url`?

Reply:
Have renamed it